com.orangedatamining.unofficial.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Orange
+Exec=/app/bin/run_orange.sh
```

```suggestion
Exec=run_orange.sh
```

it's never a full path